content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -253,6 +258,18 @@ mkdir -p /etc/systemd/system/kubelet.service.d
 curl -sSL "https://raw.githubusercontent.com/kubernetes/release/${RELEASE_VERSION}/cmd/kubepkg/templates/latest/deb/kubeadm/10-kubeadm.conf" | sed "s:/usr/bin:/opt/bin:g" > /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
 ```
 
+Add /opt/bin to PATH:
+
+```bash
+sed -i '$aexport PATH=$PATH:/opt/bin' /etc/profile&&source /etc/profile
+```
+
+Or,add /opt/bin to secure_path if you want to run the commands listed in /opt/bin with sudo:
+
+```bash
+sudo sed -i 's/secure_path.*/&\:\/opt\/bin/g' /etc/sudoers
```

Comment:
Editing sudoers outside of `visudo` or other protection is risky. I'd prefer not to include this line of advice verbatim.